src/NzbDrone.Core/Configuration/ConfigService.cs

```diff
@@ -254,32 +254,11 @@ public bool SetPermissionsLinux
 
         public string FileChmod
         {
-            get { return GetValue("FileChmod", "0644"); }
+            get { return GetValue("FileChmod", "644"); }
```

Comment:
Note: `0` prefix is used in some programming languages to indicate 'octal notation'. So that's likely why it was there.

Reply:
I changed it back.